pkg/process/process.go

```diff
@@ -0,0 +1,207 @@
+package process
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+	"time"
+
+	capnp "capnproto.org/go/capnp/v3"
+	"github.com/lthibault/log"
+	wazero_api "github.com/tetratelabs/wazero/api"
+
+	iostream_api "github.com/wetware/ww/internal/api/iostream"
+	api "github.com/wetware/ww/internal/api/process"
+	proc_errors "github.com/wetware/ww/pkg/process/errors"
+)
+
+// Process represents the execution of a function in a WASM module.
+type Process struct {
```

Comment:
We should add a `Shutdown()` method that closes the process, if it is still running.  This will prevent orphaned processes.

Capability servers with a `Shutdown()` will have that method called when the last client reference is dropped.   (See [here](https://github.com/wetware/ww/blob/feature/process/internal/api/process/process.capnp.go#L532) for entry-point into capnp.)